src/lib/sql/triggers.sql

```diff
@@ -0,0 +1,23 @@
+SELECT
+  pg_t.oid AS id,
+  is_t.trigger_schema AS schema,
+  is_t.trigger_name AS name,
+  is_t.event_manipulation AS event,
+  is_t.event_object_catalog AS catalog,
+  is_t.event_object_table AS table,
+  is_t.action_condition AS condition,
+  is_t.action_statement AS function_statement,
+  is_t.action_orientation AS orientation,
+  is_t.action_timing AS timing,
```

Comment:
@soedirgo

> The primary reason originally was performance, but without taking that into account (I don't think this SQL will be a bottleneck or anything) it's still better to only use the Postgres system catalogs directly, since infoschema is a TSQL standard (?) and often contain information that doesn't reflect Postgres internals very well

Appreciate the background and explanation!

I think we should make an exception, at least for the initial version and can go back and refactor, b/c this will get a bit complicated. `pg_trigger` only contains `tgtype` (bit mask e.g. 20) and we'll have to come up with a query like [this](https://stackoverflow.com/a/55000171).

> event_manipulation events are stored per-row (i.e. one row for each of INSERT, UPDATE, etc.) whereas pg_trigger stores them all in one row (in pg_t.tgtype). So using this, we can have rows with duplicate ids. We should instead represent this as an array.

Really good catch! I mentioned this elsewhere but I decided to just return one record and group the events into an array like you mentioned.

> I find it easier to consume if it's a function name

Agreed, I've updated PR to reflect this.

Reply:
> I think we should make an exception, at least for the initial version and can go back and refactor

Gotcha, sounds good!